lib/iron_bank/configuration.rb

```diff
@@ -8,6 +8,10 @@ class Configuration
     attr_accessor :instrumenter
     attr_accessor :instrumenter_options
 
+    # Tracing
+    attr_accessor :tracer
+    attr_accessor :tracer_options
```

Comment:
Just for my own understanding, what's the difference between a instrumenter & a tracer?

Reply:
instrumenter here is is for logging ironbank request and responses
tracer is a datadog tracer which does datadog instrumentation for each iron bank operation

probably not the best names, i am open to better naming 

reference billing PR: https://github.com/zendesk/billing/pull/4101#partial-pull-merging

@rringler

Reply:
I added a suggestion https://github.com/zendesk/billing/pull/4101 about generalizing these into a single `middlewares` configuration.